pubtools/_quay/iib_operations.py

```diff
@@ -104,11 +104,12 @@ def task_iib_add_bundles(
         tag=tag,
     )
     # Index image used to fetch manifest list. This image will never be overwritten
-    iib_namespace = build_details.index_image_resolved.split("/")[1]
```

Comment:
Looking at five nearly identical two-liners I wonder if it would make more sense to extract this snippet to a function that returns a `namedtuple`, for example.

Reply:
Good point! Added a function for it.